torch/distributed/_tensor/_dispatch.py

```diff
@@ -113,9 +116,21 @@ def dispatch(
 
         # extract local tensor and sharding infos to a OpInfo
         op_info = self.unwrap_to_op_info(op_call, args, kwargs)
+        if logger.isEnabledFor(logging.DEBUG):
+            tensor_args = ""
+            for idx, arg in enumerate(
+                itertools.chain(op_info.local_args, op_info.local_kwargs)
+            ):
+                if isinstance(arg, dtensor.DTensor):
+                    tensor_args += f"\n  arg_{idx} (dtensor): {arg.shape=} {arg._spec=}"
+                elif isinstance(arg, torch.Tensor):
+                    tensor_args += f"\n  arg_{idx} (tensor): {arg.shape=}"
```

Comment:
do we really need these? IMO shouldn't it just be:

```
logger.debug("Dispatching op_call: %s%s", op_call, op_info.op_schema)
```

Reply:
I unwrap the call because it seems to be easier to read in the log. But I can change to just print out `op_schema`.

Reply:
I think if it's not easy to read in the log, we should try to adjust `OpSchema.__str__` to make it easier to debug and read? the `__str__` is used for easier debugging purpose